frontend/src/main/java/com/google/tripmeout/frontend/TripModel.java

```diff
@@ -10,13 +10,15 @@ public static Builder builder() {
   
   public abstract String id();
   public abstract String name();
+  public abstract String userId();
```

Comment:
You might want to base this on the `double` change to avoid merge conflicts. Or you can just combine both change into a single PR. Or just keep it this way and deal with the merge conflict. ;)